frontend/src/pages/ProgressionChecker/Dashboard/Dashboard.jsx

```diff
@@ -17,6 +19,20 @@ const Dashboard = ({ isLoading, degree }) => {
     config: { tension: 80, friction: 60 },
   });
 
+  window.addEventListener("scroll", () => {
+    if (window.scrollY > 75) {
+      setarrowButton(false);
+    }
+  });
+
+  const clickArrow = () => {
```

Comment:
could we use `react-scroller` instead? (similar to the degree wizard) where instead of defining a top for scroll, we just scroll to an `id`? You can define an id value in the main page